<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreviewDialog</class>
 <widget class="QDialog" name="PreviewDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>打印预览</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="btnLayout">
     <item>
      <widget class="QToolButton" name="btnZoomIn">
       <property name="toolTip">
        <string>放大</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../tableprinterresource.qrc">
         <normaloff>:/16x16/icons/16x16/actions/viewzoomin.png</normaloff>:/16x16/icons/16x16/actions/viewzoomin.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnZoomOut">
       <property name="toolTip">
        <string>缩小</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../tableprinterresource.qrc">
         <normaloff>:/16x16/icons/16x16/actions/viewzoomout.png</normaloff>:/16x16/icons/16x16/actions/viewzoomout.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QSlider" name="hSlider">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnPrev">
       <property name="toolTip">
        <string>上一页</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../tableprinterresource.qrc">
         <normaloff>:/16x16/icons/16x16/actions/1leftarrow.png</normaloff>:/16x16/icons/16x16/actions/1leftarrow.png</iconset>
       </property>
       <property name="arrowType">
        <enum>Qt::NoArrow</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spnPage">
       <property name="toolTip">
        <string>在这里输入打印页号</string>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnNext">
       <property name="toolTip">
        <string>下一页</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../tableprinterresource.qrc">
         <normaloff>:/16x16/icons/16x16/actions/1rightarrow.png</normaloff>:/16x16/icons/16x16/actions/1rightarrow.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="btnSet">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../tableprinterresource.qrc">
         <normaloff>:/16x16/icons/16x16/actions/configure.png</normaloff>:/16x16/icons/16x16/actions/configure.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGraphicsView" name="gview"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="upLayout">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>确定</string>
       </property>
       <property name="icon">
        <iconset resource="../tableprinterresource.qrc">
         <normaloff>:/16x16/icons/16x16/actions/ok.png</normaloff>:/16x16/icons/16x16/actions/ok.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>取消</string>
       </property>
       <property name="icon">
        <iconset resource="../tableprinterresource.qrc">
         <normaloff>:/16x16/icons/16x16/actions/button_cancel.png</normaloff>:/16x16/icons/16x16/actions/button_cancel.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../tableprinterresource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>spnPage</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>800</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>726</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>PreviewDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1065</x>
     <y>579</y>
    </hint>
    <hint type="destinationlabel">
     <x>969</x>
     <y>570</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>PreviewDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1141</x>
     <y>572</y>
    </hint>
    <hint type="destinationlabel">
     <x>834</x>
     <y>577</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
